src/main/java/eu/dissco/backend/repository/MongoRepository.java

```diff
@@ -38,16 +39,16 @@ public JsonNode getByVersion(String id, int version, String collectionName)
         .get(PROV_VALUE);
   }
 
-  public List<Integer> getVersions(String id, String collectionName) throws NotFoundException {
+  public List<Integer> getVersions(String id, String collectionName, String versionProperty) throws NotFoundException {
```

Comment:
Since `versionProperty` depends on the collection, it might be a bit safer to make a map to get the version property instead of having it be passed as a method param? Just a thought

Reply:
Changed it to an Enum, I think this looks better now.